### PR TITLE
BCDA-8113: Escape export operator and fix credential rendering

### PR DESCRIPTION
**Pull request by alex-dzeda**
## 🎫 Ticket

https://jira.cms.gov/browse/BCDA-8113

## 🛠 Changes

Ensure "Try the API" credentials show properly
Escape export operator to ensure that all environments can utilize the CURL commands. 

## ℹ️ Context for reviewers

When we fixed the curl commands for windows cmd, we broke it for powershell. Now, everyone can live in harmony. 

## ✅ Acceptance Validation

image

## 🔒 Security Implications

- [ ] This PR adds a new software dependency or dependencies.
- [ ] This PR modifies or invalidates one or more of our security controls.
- [ ] This PR stores or transmits data that was not stored or transmitted before.
- [ ] This PR requires additional review of its security implications for other reasons.

If any security implications apply, add Jason Ashbaugh (GitHub username: StewGoin) as a reviewer and do not merge this PR without his approval.

### CODE REVIEW

**Review by greg-pf — APPROVED** [review]
Looks good to me

**Review by austincanada — APPROVED** [review]

**Comment by alex-dzeda** [comment]
image

File: _includes/guide/try_the_api.html [hunk 1]
```diff
@@ -11,7 +11,7 @@
 	All requests for BCDA data follow this four step process, as do other APIs following the Bulk FHIR specifications.
 </p>
 <p>
-	For more detail for each of these steps and specific information for retrieving production data, see the <a href=/build.html target="_self" class="in-text__link"> Building Your Application</a> page. For this short demo, follow along in your Terminal window or through an application such as Postman.
+	For more detail for each of these steps and specific information for retrieving production data, see the <a href=/build.html target="_self" class="in-text__link"> Building Your Application</a> page. For this short demo, follow along in your Terminal window or through an application such as Postman. Powershell users will need to replace backslash characters \ with backticks ` to properly escape the $export operation.
```
Comment (greg-pf) [inline _includes/guide/try_the_api.html:14]:
```suggestion
	For more detail for each of these steps and specific information for retrieving production data, see the <a href=/build.html target="_self" class="in-text__link"> Building Your Application</a> page. For this short demo, follow along in your Terminal window or through an application such as Postman. PowerShell users will need to replace backslash characters \ with backticks ` to properly escape the $export operation.
```

**Review by greg-pf — CHANGES_REQUESTED** [review]
Requesting a change from `Powershell` to `PowerShell`, otherwise looks good

**Review by greg-pf — APPROVED** [review]
Thanks! :shipit:

**Pull request merged**